ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -1241,7 +1241,7 @@ let migrate_send' ~__context ~vm ~dest ~live:_ ~vdi_map ~vif_map ~vgpu_map
   let suspends_vdis =
     List.fold_left
       (fun acc vm ->
-        if power_state = `Suspended then
+        if Db.VM.get_power_state ~__context ~self:vm = `Suspended then
```

Comment:
It's clear how reverting fixes the issue, maybe rename the binding to make sure it can't be mistaken again?
```suggestion
      (fun acc vm_or_snapshot ->
        let power_state = Db.VM.get_power_state ~__context ~self:vm_or_snapshot then
        if power_state = `Suspended then
```

Reply:
Let's minimize the changes at this stage :)